### ISSUE TITLE
Regarding the last issue

### ISSUE THREAD

**Issue by PrimordialHelios** [jonschlinkert/markdown-toc#133]
That was fast, unexpected, and very angry.
Seeing as I use Bitbucket and Gitlab over Github, I'm not dead, and have quite a bit of activity. Interesting that you would assume Github is the be-all end-all for programming.

Your linked document says to not ask if a project is dead, as it's very offensive for some unknown reason. It must only be offensive to you, as I've never met another person offended by that question.
I asked because this project doesn't just have a small amount of activity in the past 11 months, it has *no* activity in the past 11 months.
That's reasonable if a project is in maintenance mode, but markdown-toc has quite a few outstanding issues and PRs. It clearly has work that needs doing, and isn't in maintenance mode.

I understand not wanting to work on a project anymore, or just leaving it for a while, I've done that with several of my projects. I asked because I was planning on forking it, and I don't want to cause any issues if you intend to come back at some point.

TL;DR I asked to be polite, no reason to be rude.

So long, and thanks for all the fish.
